CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Added
+
+- Add dockerhub authentication for `kubelet`.
```

Comment:
This is a breaking change. We need to be clear about that. Otherwise dependabot will make a mess.

```suggestion
- Add dockerhub authentication for `kubelet`.

### Changed

- Add new required `DockerhubToken` parameter for templating.
```

Reply:
Ah but we also want to release it for previus majors. This sucks. We need to break the versioning then...